index.html

```diff
@@ -1,51 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Deanna Nguyen | Product Designer</title>
-    <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png"/>
-
-    <!-- BOOTSTRAP -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Code that doesn't do anything is known as 'dead' code. The files trying to be imported here between line 12 and 16 don't exist within this repository so I went ahead and removed them.